variables.tf

```diff
@@ -332,13 +332,13 @@ variable "ecs_service_platform_version" {
 variable "ecs_service_deployment_maximum_percent" {
   description = "The upper limit (as a percentage of the service's desiredCount) of the number of running tasks that can be running in a service during a deployment"
   type        = number
-  default     = 200
+  default     = 100
```

Comment:
Why do you change these defaults for ECS service? If this isn't critical for this PR which is about EFS, please revert.

Reply:
I reduced these defaults because I found that Atlantis would throw errors complaining about the BoltDB being in use when ECS would spin up the second container during ECS deployments.  The errors occurred in latest version of the container, so by reducing these defaults, ECS stops the existing container before starting the next one which avoids BoltDB being in use by two containers simultaneously.

It does come at a cost of a brief outage whenever an ECS deployment occurs.  In my testing that was always less than 5 minutes.  

I suppose we could add a local to increase these values if someone is using ephemeral storage.  Personally though, I'd rather have the outage than the lost locks in #206 which is what this PR fixes.

Reply:
Thanks for the explanation.